tomcat_deploy.bat

```diff
@@ -0,0 +1 @@
+gradle deployToTomcat
```

Comment:
1. Always add empty lines at the end of the files. U can configure your IDE to do it automatically.
2. The main intention of script files is to aggregate some logic which u don't want to type manually every time. As this file contains only 1 line there's no difference to type `gradle dT` or `tomcat_deploy.bat.`